pyproject.toml

```
@@ -40,7 +40,7 @@ deprecation = ">=2.1"
 # Documentation, must be in main dependencies (but optional) list for
 # readthedocs integration.
 # See https://github.com/readthedocs/readthedocs.org/issues/4912.
-Sphinx = {version="^4.3", optional=true}
+Sphinx = {version="^5.0", optional=true}
 sphinx_rtd_theme = {version="^1.0", optional=true}
```

Remove the upper bounds
```suggestion
Sphinx = {version=">=5.0", optional=true}
sphinx_rtd_theme = {version=">=1.0", optional=true}
```

---

Sphinx is already up to 7.2.6:
https://pypi.org/project/Sphinx/

There is a sphinx_rtd_theme 2.0:
https://pypi.org/project/sphinx-rtd-theme/